File: src/main/java/de/thatsich/intellie/common/module/AID.java
package de.thatsich.intellie.common.module;

/**
 @author thatsIch
 @since 02.04.2014. */
public abstract class AID
{
	private final String id;

	protected AID()
	{
		this.id = this.getClass().getSimpleName().toLowerCase();
	}

	@Override
	public String toString()
	{
		return this.id;
	}
}

File: src/main/scala/de/thatsich/minecraft/intellie/common/AID.scala
package de.thatsich.minecraft.intellie.common

/**
 *
 *
 * @author thatsIch
 * @since 07.04.2014.
 */
/**
@author thatsIch
@since 02.04.2014. */
abstract class AID
{
	private final val id: String = this.getClass.getSimpleName.toLowerCase

	override def toString: String =
	{
		this.id
	}
}